Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -881,6 +881,13 @@ def addCKFTracks(
 
     s.addWhiteboardAlias("trajectories", trackFinder.config.outputTrajectories)
 
+    s.addAlgorithm(
+        acts.examples.TrajectoryPrinter(
+            level=customLogLevel(),
+            inputTrajectories=trackFinder.config.outputTrajectories,
+        )
+    )
```

Comment:
should we hide this behind a flag or just have it switching on with the correct debug level?

Reply:
Hm, could do I guess.